node/pkg/logscribeconsumer/app.go

```diff
@@ -0,0 +1,220 @@
+package logscribeconsumer
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"time"
+
+	errorsentinel "bisonai.com/orakl/node/pkg/error"
+	"bisonai.com/orakl/node/pkg/utils/request"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func New(options ...AppOption) (*App, error) {
+	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
+
+	c := &AppConfig{
+		StoreInterval:     DefaultLogStoreInterval,
+		Buffer:            DefaultBufferSize,
+		LogscribeEndpoint: DefaultLogscribeEndpoint,
+		Level:             DefaultMinimalLogStoreLevel.String(),
+	}
+	for _, option := range options {
+		option(c)
+	}
+
+	resp, err := request.RequestRaw(request.WithEndpoint(c.LogscribeEndpoint))
+	if err != nil || resp.StatusCode != http.StatusOK {
+		return nil, errorsentinel.ErrLogscribeConsumerEndpointUnresponsive
+	}
+
+	if c.Service == "" {
+		log.Error().Msg("Service not provided")
+		return nil, errorsentinel.ErrLogscribeConsumerServiceNotProvided
+	}
+	level, err := zerolog.ParseLevel(c.Level)
+	if err != nil {
+		log.Error().Msgf("Error parsing log level, falling back to default value: %s", DefaultMinimalLogStoreLevel.String())
```

Comment:
let's remove this error log, or set this as debug level. don't think it is significant to be set as error log